README.md

```diff
@@ -1,4 +1,96 @@
 # Paloma
 
+## Setting up a new chain from scratch.
```

Comment:
could you add this after those badges?

Reply:
Ah that's what that is. Definitely. I also need to rebase on @taariq's changes after https://github.com/palomachain/paloma/pull/133 and https://github.com/palomachain/paloma/pull/134 are merged.